Sources/CoreLocationMiddleware/CoreLocationMiddleware.swift

```diff
@@ -141,29 +179,69 @@ public final class CoreLocationMiddleware: Middleware {
     }
 }
 
+// MARK: - DELEGATE
 class CLDelegate: NSObject, CLLocationManagerDelegate {
     
     var output: AnyActionHandler<LocationAction>? = nil
-    
+
+    @available(iOS 14.0, *)
+    func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
+        output?.dispatch(
+            LocationAction.status(
+                getAuthzStatus(
+                    status: manager.authorizationStatus,
+                    accuracy: manager.accuracyAuthorization
+                )
+            )
+        )
+    }
+
```

Comment:
This seems to work... just not sure if I am doing the *right* thing here to compartmentalize those 2 delegate methods (pre and post iOS 14).
Note that from the tests in iOS 14, only `locationManagerDidChangeAuthorization` is being called (so that's good, right ? 😬)